pkg/ui/src/views/shared/components/alertMessage/alertMessage.tsx

```diff
@@ -0,0 +1,94 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+import React from "react";
```

Comment:
I'm confused as to which new components go in `src/views/shared/components` and which ones in `/src/components`? I assumed all new work would go in the second directory.

Reply:
@dhartunian , I also thought it has to be in `/src/components`, but this component is implemented in such a way to be compatible (reuse the same interface as another AlertBox component).
This required to use `react-router-dom` lib and reference to `import { AlertInfo, AlertLevel } from "src/redux/alerts"` inside component.
With all of this dependencies it doesn't look like pure component so I decided to put in shared components.